chia/rpc/data_layer_rpc_api.py

```diff
@@ -11,6 +12,8 @@
 # todo input assertions for all rpc's
 from chia.util.streamable import recurse_jsonify
 
+log = logging.getLogger(__name__)
```

Comment:
Don't we generally pass in loggers rather than creating them locally?  Also, the changes here look all unused?